runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -267,8 +267,6 @@ static const char* getNameForMethodRelocation (int type)
             return "TR_VirtualRamMethodConst";
          case TR_ClassAddress:
             return "TR_ClassAddress";
-         case TR_StaticDefaultValueInstance:
```

Comment:
The reference to [`isStaticDefaultValueInstance()`](https://github.com/eclipse-openj9/openj9/blob/ba1f008911bce0ac1d5d254c0eb97836a3c9b7cb/runtime/compiler/il/J9SymbolReference.cpp#L330-L333) in J9SymbolReference.cpp can be removed